packages/app-plugin-admin-welcome-screen/src/components/Welcome.tsx

```diff
@@ -112,10 +122,18 @@ const Welcome = () => {
                     <SimpleFormContent>
                         <ContentTheme>
                             <Cell span={12}>
-                                <Typography use={"headline6"}>
-                                    <p className={pGetStartedStyle}>
-                                        To get started - pick one of the actions below:
-                                    </p>
+                                <Typography use={"headline6"}>      
+                                    <h6>{getPlugins<AdminWelcomeScreenWidgetPlugin>("admin-welcome-screen-widget")}</h6>
```

Comment:
@TommyJackson85 this line here is causing your React error. You're trying to render an array of objects. Not sure what you wanted to render here, but this is the place to fix.